src/app/CommandHandlerImpl.cpp

```diff
@@ -420,6 +410,17 @@ Status CommandHandlerImpl::ProcessCommandDataIB(CommandDataIB::Parser & aCommand
         }
     }
 
+    {
+        Status commandExists = mpCallback->CommandExists(concretePath);
```

Comment:
Followup: the spec has the fabric-scoped check before the existence check, whereas we have it after the timed invoke check.  I actually think the spec is wrong, because for commands that don't exist, how is the server supposed to know whether they are fabric-scoped?  We should fix the spec to do that check right after the existence checks, and then update this code.